### DISCUSSION TITLE
gongju/choco

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#156] [Announcements]
# gongju/choco

chocolatey：一款 GitHub 星标 8.2k+ 的 Windows 命令行软件管理神器，好用到爆！

https://tobebetterjavaer.com/gongju/choco.html

**Comment by zisoqlsdha** [comment]
'Set-ExecutionPolicy' 不是内部或外部命令，也不是可运行的程序
或批处理文件。
为啥出现这个问题

**Reply by QMZCLL** [reply]
用PowerShell
